spring-ws-core/src/main/java/org/springframework/ws/server/endpoint/mapping/PayloadRootQNameEndpointMapping.java

```diff
@@ -43,9 +43,7 @@
  * @author Arjen Poutsma
  * @see org.springframework.xml.namespace.QNameEditor
  * @since 1.0.0
- * @deprecated as of Spring Web Services 2.0, in favor of {@link PayloadRootAnnotationMethodEndpointMapping}.
```

Comment:
Regarding @see, good point.

I chatted with @poutsma about docs. Essentially, Spring MVC doesn't document the older way of doing things, and we essentially follow Spring MVC. So we aren't bringing back the doc stuff.